dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/command/DownloadCommand.java

```diff
@@ -68,8 +68,8 @@
   private boolean force = false;
   @Parameter(names = "--manifest", description = "Path to manifest id, url or file")
   private ManifestResource manifestResource;
-  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdValidator.class)
-  private String objectId;
+  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdListValidator.class, variableArity = true)
```

Comment:
Are you sure the previous version wouldn't work with `variableArity = true`? You shouldn't have to do anything special with lists.